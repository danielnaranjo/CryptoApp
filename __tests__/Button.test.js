import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../app/components/Button';

it('Button renders correctly', () => {
  const tree = renderer.create(<Button title="TEST" onPress={() => console.log('BUTTON PRESSED')} />);
  expect(tree).toMatchSnapshot();
});
