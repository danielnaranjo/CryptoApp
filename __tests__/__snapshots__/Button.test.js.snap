// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "grey",
          "marginBottom": 10,
          "marginTop": 10,
          "paddingBottom": 10,
          "paddingLeft": 40,
          "paddingRight": 40,
          "paddingTop": 10,
          "width": 600,
        },
        Object {},
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 14,
          "textAlign": "center",
        }
      }
    >
      TEST
    </Text>
  </View>
</View>
`;
